src/less/widgets/slideouts.less

```diff
@@ -32,6 +32,10 @@
     font-size: 20px;
   }
 
+  .pf-widget-footer {
+    margin-top: 10px;
```

Comment:
see below

Reply:
@markhayden I already checked that nothing is visually different about the modals/slideouts if there is no footerText setting.  I had it as padding-top at first which did change the appearance of the modal since it created a div with a height of 10, but margin didn't have that effect.

Reply:
perfect, just making sure